_ext/jellyfin.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: ext
+title: Mopidy-Jellyfin
+logo: /media/ext/jellyfin.png
+type: backend
+service: Jellyfin
```

Comment:
This is only used when there's an oauth section so we don't need this.

Reply:
I'm assuming you mean just the `service` line?

Reply:
Yep, sorry, should have been clearer.